deis-tests/manifests/deis-tests-pod.yaml

```diff
@@ -10,5 +10,5 @@ spec:
   - name: deis-e2e
     image: quay.io/deisci/deis-e2e:canary
     command: ["/bin/sh", "-c"]
-    args: ["/bin/tests.test || true"]
+    args: ["/bin/tests.test"]
```

Comment:
It may be vestigial from when I tried to implement the chart with a Kubernetes [Job](http://kubernetes.io/v1.1/docs/user-guide/jobs.html), which won't take no for an answer and doesn't respect `restartPolicy=never`. If it works without `|| true`, then great--I will go change [the tests chart](https://github.com/deis/charts/tree/master/deis-tests).